recipe/meta.yaml

```diff
@@ -1,14 +1,18 @@
 {% set name = "qcdloop" %}
-{% set version = "2.0.11" %}
+{% set version = "2.1.0" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/scarrazza/{{ name }}/archive/{{ version }}.tar.gz
-  sha256: a6274a4f552209c5e8c41d265af52214ac72a235a25d3b771f969deb2f15788a
+  sha256: 75ac2d00231714210859d39c4beebce14f183707afe47f4550f33ef9f348b3c1
   folder: source
+  patches:
+    # Remove in releases after v2.1.0
+    # c.f. https://github.com/scarrazza/qcdloop/pull/35
```

Comment:
This patch can get removed in any release following `v2.1.0`. c.f. https://github.com/scarrazza/qcdloop/pull/35